Makefile

```diff
@@ -31,7 +31,7 @@ test-all:
 	tox
 
 build-docs:
-	sphinx-apidoc -o docs/ . setup.py "eth_abi/utils/*" "*conftest*" "tests/*"
+	sphinx-apidoc -o docs/ . setup.py 'eth_typing/*' 'eth_abi/encoding.py' 'eth_abi/decoding.py' 'eth_abi/utils/*' 'tests/*'
```

Comment:
Why not generate API documentation for encoding & decoding?

Reply:
I guess we don't need to exclude those modules.  I think I did that because I was trying to find a way to exclude certain things from being included in the API documentation.  But then I realized I could restrict things from showing up with the `:members:` option on the `automodule` directive.  I think I can revert this change.